apps/native/app/src/screens/document-detail/utils/share-file.tsx

```diff
@@ -18,11 +18,15 @@ export const shareFile = ({ document, pdfUrl }: ShareFileProps) => {
     authStore.setState({ noLockScreenUntilNextAppStateActive: true })
   }
 
-  Share.open({
-    title: document.subject,
-    subject: document.subject,
-    message: `${document.sender.name} \n ${document.subject}`,
-    type: pdfUrl ? 'application/pdf' : undefined,
-    url: pdfUrl ? `file://${pdfUrl}` : document.downloadUrl!,
-  })
+  try {
+    await Share.open({
+      title: document.subject,
+      subject: document.subject,
+      message: `${document.sender.name} \n ${document.subject}`,
+      type: pdfUrl ? 'application/pdf' : undefined,
+      url: pdfUrl ? `file://${pdfUrl}` : document.downloadUrl!,
+    })
+  } catch (error) {
+    // noop
```

Comment:
No sure what we want to do here? If the user decides to not share the document, i.e. clicks share and then x on the share modal, then it is an error from the package `[Error: User did not share]`. Should we swallow that error but display others? Not sure 😅 We don't have any good workflow for errors in the app yet 🫠